packages/react-devtools-shared/src/backend/agent.js

```diff
@@ -225,6 +225,9 @@ export default class Agent extends EventEmitter<{|
       bridge.send('profilingStatus', true);
     }
 
+    // Send the Bridge protocol after initialization in case the frontend has already requested it.
+    this._bridge.send('bridgeProtocol', currentBridgeProtocol);
```

Comment:
Made this change to avoid potential timing problems between the backend and frontend initialization sequence. (If the frontend asked the backend for a protocol _before_ the backend was listening, it would cause the frontend to timeout waiting and show a modal:
![Screen Shot 2021-12-14 at 9 17 31 AM](https://user-images.githubusercontent.com/29597/146015475-50d55738-aa6c-4ccd-b6a9-c12688d5c627.png)

Reply:
does this change affect the extension or standalone apps in any way? should we test that?

Reply:
I tested the extension. This will only affect RN though.